Mathlib/RingTheory/Coalgebra/Basic.lean

```diff
@@ -37,6 +37,29 @@ class CoalgebraStruct (R : Type u) (A : Type v)
   /-- The counit of the coalgebra -/
   counit : A →ₗ[R] R
 
+/--
+A representation of an element `a` of a coalgebra `A` is a finite sum of pure tensors `∑ xᵢ ⊗ yᵢ`
+that is equal to `comul a`.
+-/
+structure CoalgebraStruct.Repr (R : Type u) {A : Type v}
```

Comment:
I might be useful to keep `Repr` independent from the other axioms, so that `Repr` can be used in constructing say tensor product of coalgebras before we prove the axioms for tensor product of coalgebra

Reply:
I'm not suggesting changing the type, just the namespace.

Reply:
This has been implemented throughout! Thanks for the suggestion